### ISSUE TITLE
fix toggle menu button

### ISSUE THREAD

**Issue by criticalbh** [criticalbh/material2-admin#1]
toggle menu is not working when screen is in extra large size

**Comment by esouthren** [comment]
I'll give this a try

**Comment by esouthren** [comment]
Just to clarify, you'd like the menu to toggle visible/invisible on desktop as well as mobile?

**Comment by criticalbh** [comment]
Right, mobile version should use sidenav mode over, and desktop should use push.

**Comment by esouthren** [comment]
I'm struggling to get a live preview for this for development, even when I host my forked repo on Github pages. Can you help on this?

**Comment by criticalbh** [comment]
Of course, I did not work with github pages so I am not familiar, but to start locally with development you need to have `yarn`. 

If you have shell, after cloning the repository, inside project directory run:
* `$ yarn` -> dependency installation
* `$ yarn start` -> open localhost:8001 to preview the app.


For live reload you should use angular-cli. 
After `yarn` command is executed successfully, start development server with
`$ ./node_modules/.bin/ng serve` (this does not enforce you to install angular-cli on global level).

If there are any other questions please do not hesitate to ask.
